tardis/energy_input/gamma_packet_loop.py

```diff
@@ -103,21 +101,23 @@ def gamma_packet_loop(
     escaped_packets = 0
     scattered_packets = 0
     packet_count = len(packets)
+    # Logging does not work with numba
     print("Entering gamma ray loop for " + str(packet_count) + " packets")
 
-    deposition_estimator = np.zeros_like(energy_df_rows)
-
     for i in range(packet_count):
         packet = packets[i]
         time_index = get_index(packet.time_current, times)
+        energy_deposited_positron[
+            packet.shell, time_index
+        ] += packet.positron_energy
 
```

Comment:
Set positron energy to zero here now that it has been deposited?

Reply:
Alternatively, remove and calculate separately.

Reply:
i.e. instead of in this loop, use the dataframe and sum all the positron energy per shell per time step and use those to fill out this array

Reply:
I removed the positron energy deposition calculation out of the loop.